src/libs/actions/IOU.ts

```diff
@@ -314,7 +314,7 @@ function clearMoneyRequest(transactionID: string) {
 /**
  * Update money request-related pages IOU type params
  */
-function updateMoneyRequestTypeParams(routes: StackNavigationState<ParamListBase>['routes'] | NavigationPartialRoute[], newIouType: string, tab: string) {
+function updateMoneyRequestTypeParams(routes: StackNavigationState<ParamListBase>['routes'] | NavigationPartialRoute[], newIouType: string, tab?: string) {
```

Comment:
Are we sure making `tab?` won't have any impact?

Reply:
In this specific case we had to make it optional as it was never passed a tab prop.
I've tested the other calls of the functions and everything was working as expected.